uber/templates/regform.html

```diff
@@ -96,12 +96,17 @@
         <div class="col-sm-6">
             <input type="text" name="last_name" id="last_name" value="{{ attendee.last_name }}" class="form-control" placeholder="Last Name">
         </div>
-        {% if c.PAGE_PATH != '/registration/form' %}
-            <p class="help-block col-sm-6 col-sm-offset-2">
-                These must be your legal name.
-                <span class="popup">{% popup_link "../static_views/legal_name.html" "What does this mean?" %}</span>
-            </p>
-        {% endif %}
+    </div>
+
+    <div class="form-group">
```

Comment:
@kitsuta you've been in charge of registration more recently than me so please chime in on this:

I think it would be better to *not* display the Legal Name by default and to instead put a checkbox next to First and Last Name that says something like "This is not my legal name", and checking that checkbox causes the Legal Name row to appear.  People are going to instinctively tab through the form fields and just fill out everything without reading the instructions because people don't read instructions.  If we have a field which someone only cares about if they want to make it different, then we should explicitly have a checkbox which shows/hides that row.